internal/runners/clean/run_win.go

```diff
@@ -22,24 +22,36 @@ import (
 )
 
 func (u *Uninstall) runUninstall() error {
+	// we aggregate installation errors, such that we can display all installation problems in the end
```

Comment:
I know this change is not in scope for this PR. I realized that the behavior was inconsistent between Mac/Linux and Windows.  I am pretty sure that we want it this way, but I am happy to take it out and create a separate story for this.